src/types/redis_bitmap_string.h

```diff
@@ -56,6 +57,9 @@ class BitmapString : public Database {
   // Return:
   //  The normalized [start, end] range.
   static std::pair<int64_t, int64_t> NormalizeRange(int64_t origin_start, int64_t origin_end, int64_t length);
+
+  static std::pair<int64_t, int64_t> AdjustMaskWithRange(bool is_bit, int64_t origin_start, int64_t origin_end,
```

Comment:
Can you add a comment to tell others what this function does?

Reply:
Something like "normalized bit to byte range" would help